spec/vehicle_factory_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+
+RSpec.describe VehicleFactory do
+  describe '#create_vehicles' do
+    it 'returns an array' do
+      factory = VehicleFactory.new
+      vehicles = factory.create_vehicles([])
+
+      expect(vehicles).to be_an_instance_of(Array)
+    end
+
+    it 'returns an empty array when given an empty data set' do
+      factory = VehicleFactory.new
+      vehicles = factory.create_vehicles([])
```

Comment:
Make sure you also add a test for when you send in a dataset with real data. It doesn't have to be the full dataset, but could be an array of 2 sample hashes, and you could just verify that 2 Vehicle objects get created.